tests/ctst/features/iam-policies/IAMUser.feature

```diff
@@ -0,0 +1,82 @@
+Feature: IAM Policies for IAM Users
+  This feature allows you to create and attach IAM policies for IAM users.
+  IAM users should have the permissions to perform the actions that they are granted in their IAM policies.
+
+  @2.6.0
+  @PreMerge
+  @IAM-Policies-IAMUsers
+  Scenario Outline: User is not authorized to perform the actions without IAM policy
+    Given an existing bucket "<bucketName>" "without" versioning
+    And we "<haveObject>" an existing object "<objectName>" "without" version-Id ""
+    And a IAM_USER type
+    When the user tries to perform "<ifS3Standard>" "<action>" on the bucket "<haveObject>" the object
+    Then the user should receive an access denied error
+
+    Examples:
+      | action         | ifS3Standard  | bucketName | haveObject | objectName |
+      | MetadataSearch | notS3Standard |            | havenot       |            |
+      | RestoreObject  | S3Standard    |            | have    |            |
+
+  @2.6.0
+  @PreMerge
+  @IAM-Policies-IAMUsers
+  Scenario Outline: User is authorized to perform the actions that are granted in the IAM policy
+    Given an existing bucket "<bucketName>" "without" versioning
+    And we "<haveObject>" an existing object "<objectName>" "without" version-Id ""
+    And a IAM_USER type
+    And an existing IAM policy with "<effect>" effect to perform "<action>" on "<resource>"
+    And attach the IAM policy to the user
+    When the user tries to perform "<ifS3Standard>" "<action>" on the bucket "<haveObject>" the object
+    Then the user should be able to perform successfully
```

Comment:
We should be more explicit about what is expected, in the sentence:

```suggestion
    Then the user should be able to perform successfully the "<action>" action
```

Reply:
updating in common PR, starting here ed5d2bd2e26929a6a1decb7086b79ade19e1e988